docker-compose/docker-compose.yaml

```
@@ -0,0 +1,8 @@
+services:
+    betterbahn:
+        build:
```

Building locally is something I'd recommend moving to a compose.dev.yml.

```suggestion
        image: ghcr.io/l2xu/betterbahn:testing-main
```